src/views/Entries/FilterEntriesForm/index.js

```diff
@@ -379,6 +395,25 @@ export default class FilterEntriesForm extends React.PureComponent {
                     disabled={pending}
                     placeholder={_ts('entries', 'commentStatusPlaceholder')}
                 />
+                <SelectInput
+                    className={styles.entriesFilter}
+                    keySelector={FilterEntriesForm.optionKeySelector}
+                    labelSelector={FilterEntriesForm.optionLabelSelector}
+                    options={verificationStatusOptions}
+                    label={_ts('entries', 'verificationStatusOptionsFilterLabel')}
+                    onChange={(value) => {
+                        const option = verificationStatusOptions.find(v => v.key === value);
+                        const status = option ? option.isVerified : undefined;
+                        this.handleFilterChange(
+                            'verified',
+                            status,
+                        );
+                    }}
```

Comment:
Why no use `verified` as keySelector? Does SelectInput not support it?
We can use SegmentInput here btw!

Reply:
SelectInput doesnot support boolean as key

Reply:
Lets not use SegmentInput, it also has to support 'none/all'. Lets use SelectInput